source/globalCommands.py

```diff
@@ -1956,6 +1961,70 @@ def script_reportOrShowFormattingAtCaret(self, gesture):
 		elif repeats == 1:
 			self.script_showFormattingAtCaret(gesture)
 
+	@script(
+		description=_(
+			# Translators: the description for the reportDetailsSummary script.
+			"Report summary of any annotation details for this navigator position."

```

Comment:
I feel this is quite odd to report at a position of the review cursor since in both browse and focus modes you're navigating with a caret rather than with the review cursor. Has it been considered to report the details at a position of the browse mode caret if it exists and fall back to the focused object if there is no caret or user is in focus mode?

Reply:
Are you referring the behavior or just this description of the behavior. The description could certainly be more precise.

If you are referring to the behavior, could you please give an example that highlights how it is odd?

Reply:
Ive been referring to the behavior. My main problem with using position of the review cursor is that in both browse mode and content editable you're navigating with system focus or system caret, and while by default review cursor and navigator object tracks them this is optional. This means that:
1. If you're working with review cursor not following caret most of the time you need to move it from wherever you've  left it just to check annotations at your current focus / caret position and 
2. Even if you're working with the default configuration for the review cursor most of the time you may want to disable following of the navigator object for a single task, forgot about it and be caught by surprise when you want to check for annotations.

Reply:
I believe this is now addressed.